resources/index.html

```diff
@@ -5,7 +5,7 @@
   </head>
   <body>
     <h2>APIs.guru REST API</h2>
-    <p>This root for <a href="http://APIs.guru">APIs.guru</a> API, please read details there.<br>Here is list of related links:</p>
+    <p>This is the root for <a href="http://APIs.guru">APIs.guru</a> API, More to be updated later. <br>Here is list of related links:</p>
```

Comment:
By "more to be updated later" do you mean you're going to add to this PR, or something else?